mappings/net/minecraft/entity/ai/brain/task/Task.mapping

```diff
@@ -33,7 +36,8 @@ CLASS ajj net/minecraft/entity/ai/brain/task/Task
 		ARG 1 world
 		ARG 2 entity
 		ARG 3 time
-	METHOD f stopInternal (Lve;Laii;J)V
+	METHOD f finishRunning (Lve;Laii;J)V
+		ARG 3 time
```

Comment:
`finishExecution` feels a bit more natural, but maybe that's just me.

Reply:
In that case, how's `shouldExecute`, `execute`, `shouldContinueExecution`, `continueExecution`, and `finishExecution`? I'd like to keep the lifecycle names consistent with each other.

Reply:
I would personally be in favour of that change, however `Goal`s use `run`, not execute. As such it may be better to stay consistent and keep `*running`.

Reply:
I'll leave it as "running" then.